src/components/KInput/KInput.vue

```diff
@@ -245,6 +247,19 @@ const getValue = (): string | number => {
   return currValue.value || modelValueChanged.value ? currValue.value : modelValue
 }
 
+const focus = () => {
+  inputRef.value?.focus?.()
+}
+
+const blur = () => {
+  inputRef.value?.blur?.()
+}
+
+defineExpose({
+  focus,
+  blur,
+})
```

Comment:
suggestion: can we just expose the input ref rather than focus and blur methods?